apps/site/pages/docs/getting-started/2-setting-up-the-project.mdx

```diff
@@ -89,14 +93,21 @@ Before cloning your store's repository, ensure that you have the following tools
   since our documentation may include screenshots from VS Code.
 </Callout>
 
+### Enabling Developer Mode (Windows users)
+We strongly recommend enabling Developer Mode when using the terminal for FastStore projects. 
+Since FastStore projects rely on creating symlinks, this mode grants the necessary permissions and privileges, reducing the chance of errors during development. 
+To activate Developer Mode, refer to Microsoft's official guide on [Enabling your device for development](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers).
```

Comment:
```suggestion
### Enabling Developer Mode (Windows users)
We strongly recommend enabling Developer Mode when using the terminal for FastStore projects. 
Since FastStore projects rely on creating symlinks, this mode grants the necessary permissions and privileges, reducing the chance of errors during development. 
To activate Developer Mode, refer to Microsoft's official guide on [Enabling your device for development](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers).

<Callout type="warning" emoji="⚠️">
  Running the FastStore project as an Administrator is not recommended.
</Callout>

```